<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>980</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTipDuration">
   <number>2</number>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:'#3c4043'; color:'white';
</string>
  </property>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>90</y>
     <width>512</width>
     <height>512</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="start_button">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>790</y>
     <width>421</width>
     <height>91</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777177</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:30px;
color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 #84E9FF , stop:1 #C284FF);
</string>
   </property>
   <property name="text">
    <string>시작하기</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1610</x>
     <y>910</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>종료하기: ESC</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
